zhaquirks/schneiderelectric/smoke.py

```diff
@@ -0,0 +1,174 @@
+        se_no_disturb_mode: Final = ZCLAttributeDef(
+            id=0x0006,
+            type=t.Bool,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+        se_cluster_revision: Final = ZCLAttributeDef(
+            id=0xFFFD,
+            type=t.uint16_t,
+            access="r",
+            is_manufacturer_specific=True,
+        )
+
+
+(
+    QuirkBuilder(SE_MANUF_NAME, "W599001")
+    .applies_to(SE_MANUF_NAME, "W599501")
+    .applies_to(SE_MANUF_NAME, "755WSA")
+    .replaces(SEBasic, endpoint_id=20)
+    .replaces(SEAlarmConfiguration, endpoint_id=20)
+    .enum(
+        cluster_id=SEAlarmConfiguration.cluster_id,
+        endpoint_id=20,
+        attribute_name=SEAlarmConfiguration.AttributeDefs.se_led_brightness.name,
+        translation_key="led_brightness",
+        fallback_name="LED brightness",
+        enum_class=SELEDBrightness,
+    )
+    .enum(
+        cluster_id=SEAlarmConfiguration.cluster_id,
+        endpoint_id=20,
+        attribute_name=SEAlarmConfiguration.AttributeDefs.se_alarm_sound_level.name,
+        translation_key="alarm_sound_level",
+        fallback_name="Alarm sound level",
+        enum_class=SEAlarmSoundLevel,
+    )
+    .enum(
+        cluster_id=SEAlarmConfiguration.cluster_id,
+        endpoint_id=20,
+        attribute_name=SEAlarmConfiguration.AttributeDefs.se_alarm_sound_mode.name,
+        translation_key="alarm_sound_mode",
+        fallback_name="Alarm sound mode",
+        enum_class=SEAlarmSoundMode,
+    )
+    .sensor(
+        cluster_id=SEAlarmConfiguration.cluster_id,
+        endpoint_id=20,
+        attribute_name=SEAlarmConfiguration.AttributeDefs.se_lifetime.name,
+        translation_key="lifetime",
+        fallback_name="Lifetime",
+        device_class=NumberDeviceClass.DURATION,
+        unit=UnitOfTime.YEARS,
+        multiplier=0.5,
+        entity_type=EntityType.DIAGNOSTIC,
+    )
+    .number(
+        cluster_id=SEAlarmConfiguration.cluster_id,
+        endpoint_id=20,
+        attribute_name=SEAlarmConfiguration.AttributeDefs.se_hush_duration.name,
+        translation_key="hush_duration",
+        fallback_name="Hush duration",
+        device_class=NumberDeviceClass.DURATION,
+        unit=UnitOfTime.MINUTES,
+        min_value=5,
+        max_value=15,
+        step=1,
+    )
+    .switch(
+        cluster_id=SEAlarmConfiguration.cluster_id,
+        endpoint_id=20,
+        attribute_name=SEAlarmConfiguration.AttributeDefs.se_test_mode.name,
+        translation_key="test_mode",
+        fallback_name="Test mode",
+        reporting_config=ReportingConfig(
+            min_interval=15,
+            max_interval=300,
+            reportable_change=1,
+        ),
+    )
+    .switch(
+        cluster_id=SEAlarmConfiguration.cluster_id,
+        endpoint_id=20,
+        attribute_name=SEAlarmConfiguration.AttributeDefs.se_no_disturb_mode.name,
+        translation_key="no_disturb_mode",
+        fallback_name="No disturb mode",
```

Comment:
Can we call this "Do not disturb mode" or "Silence alarm" or something like that?
"No disturb mode" sounds a bit like something is missing or wrong grammatically

Reply:
Fixed